src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java

```diff
@@ -61,16 +61,17 @@
         caseData.getIssueDate() != null && caseData.getRespondent1Represented() == NO;
 
     public static final Predicate<CaseData> respondent1OrgNotRegistered = caseData ->
-        caseData.getIssueDate() != null && caseData.getRespondent1OrgRegistered() == NO;
+        caseData.getIssueDate() != null
+            && caseData.getRespondent1OrgRegistered() == NO
+            && caseData.getRespondent1Represented() == YES;
 
     public static final Predicate<CaseData> respondent2NotRepresented = caseData ->
         caseData.getIssueDate() != null && caseData.getRespondent2Represented() == NO;
 
     public static final Predicate<CaseData> respondent2OrgNotRegistered = caseData ->
         caseData.getIssueDate() != null
             && caseData.getRespondent2Represented() == YES
-            && caseData.getRespondent2OrgRegistered() != YES
-            && caseData.getRespondent2SameLegalRepresentative() != YES;
```

Comment:
I don't think that this is right.

In a case where:
Def1 solicitor is not registered and Def2 has the same solicitor, respondent2OrgNotRegistered should still be true.
I've removed it now

Reply:
agreed, not sure of the relevance for `caseData.getRespondent2SameLegalRepresentative() != YES;` in `respondent2OrgNotRegistered` Predicate

There seems to be no risk to mutual eclusion